packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -430,6 +457,12 @@ class FlutterErrorDetails extends Diagnosticable {
         ));
       }
     }
+    if (offendingContext != null && offendingContextDescribers.isNotEmpty) {
+      properties.add(ErrorSpacer());
+      for(FlutterOffendingContextDescriber describer in  offendingContextDescribers) {
+        describer(offendingContext).forEach(properties.add);
```

Comment:
maybe add the ErrorSpacer before each one, so they are separated from each other?